crates/holochain/src/conductor/conductor.rs

```diff
@@ -421,7 +421,7 @@ mod interface_impls {
                             allowed_origins,
                         } => {
                             let listener = spawn_websocket_listener(port, allowed_origins).await?;
-                            let port = listener.local_addr()?.port();
+                            let port = listener.local_addrs()?[0].port();
```

Comment:
Prefer not to do unchecked array indexing but it doesn't make sense to have a listener with no addresses so I think it's reasonable in this case